docs/3.api/1.components/12.google-maps.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "<GoogleMaps>"
+description: A simple and performant Google Maps component.
+links:
+  - label: Source
+    icon: i-simple-icons-github
+    to: https://github.com/nuxt/scripts-and-assets/blob/main/modules/nuxt-third-party-capital/src/runtime/components/GoogleMaps.ts
+    size: xs
+---
+
+The GoogleMaps component feels like a Google Maps Embed, but uses the [Maps JavaScript API](https://developers.google.com/maps/documentation/javascript) to load and initialize the map.
+
+::callout
+You need an api key to use Google Maps.
+::
+
+## Params
+
+An object containing the following options:
+
+| name | type | description                       |
+|:-----|:-------|:--------------------------------|
+| apiKey | string | The [api key](https://developers.google.com/maps/documentation/javascript/get-api-key) needed to use Google Maps API. (required)|
+| q | string | A query to search for. (optional)|
+| center | LatLng | Coordinates to set the map to. (optional) |
+| zoom | number | Zoom value for the map. (optional) |
+| width | string | Width of the player. (optional)|
+| height | string | Height of the player. (optional)|
+
```

Comment:
Are there any additional properties not specified here but, if they get included, get passed on? If so, maybe including a note saying something like "For a full list of options, read the Google Map JS API docs..." with a link.

If there are no other props that can be included, then ignore this :)